src/app/app-routing.module.ts

```diff
@@ -18,7 +19,8 @@ const routes: Routes = [
   { path: 'device-creation', component: DeviceCreationComponent },
   { path: 'network', component: NetworkComponent },
   { path: 'wiki', component: WikiComponent },
-  { path: 'wiki-page/:id', component: WikiPageComponent}
+  { path: 'wiki-page/:id', component: WikiPageComponent},
+  { path: 'profile-page/:userName', component: ProfilePageComponent}
```

Comment:
I don't think that this path is a good idea since the names of the users don't need to be unique. For now, I would rather suggest using something like `/my-profile` or `/settings/profile`.